src/bench/crypto_hash.cpp

```diff
@@ -105,6 +106,70 @@ static void FastRandom_1bit(benchmark::Bench& bench)
     });
 }
 
+static void MuHash(benchmark::Bench& bench)
+{
+    FastRandomContext rng(true);
+    MuHash3072 acc;
+    unsigned char key[32] = {0};
+    int i = 0;
+    bench.run([&] {
+        key[0] = ++i;
+        acc *= MuHash3072(key);
+    });
+}
+
+static void MuHashAdd(benchmark::Bench& bench)
+{
+    FastRandomContext rng(true);
+    MuHash3072 acc;
+    unsigned char key[32];
+    std::vector<unsigned char> randkey = rng.randbytes(32);
+    for (size_t i = 0; i < randkey.size(); ++i) {
+        key[i] = randkey[i];
+    }
+
+    MuHash3072 muhash = MuHash3072(key);
```

Comment:
The temporary array and vector variables are unnecessary here. The MuHash constructor takes a span, so we can just call it directly with the rvalue vector from randbytes().

```suggestion
    MuHash3072 acc;
    FastRandomContext rng(true);
    MuHash3072 muhash{rng.randbytes(32)};

```

Same in the other functions below.

Reply:
Done